dashboard/app/controllers/openai_chat_controller.rb

```diff
@@ -1,7 +1,9 @@
 class OpenaiChatController < ApplicationController
-  include OpenaiChatHelper
   authorize_resource class: false
 
+  API_KEY = CDO.openai_chat_completion_api_key
```

Comment:
We'll create a new "Project" for AI Tutor, and add a new key there (within the Code.org account), and replace this value.

@ebeastlake shared a screenshot of what looked like a separate Account ("Code.org AI Tutor" I think, per the screenshot in the Slack mentioned linked below) for AI Tutor where she noted that the key had never been used. I think we can delete that Account, as Seth confirmed that we should be creating Projects within the main Code.org account.

https://codedotorg.slack.com/archives/C06UT1E2796/p1737747972399459?thread_ts=1737744510.763629&cid=C06UT1E2796